pkgs/development/python-modules/docutils/default.nix

```diff
@@ -17,7 +17,7 @@ buildPythonPackage rec {
 
   # Only Darwin needs LANG, but we could set it in general.
   # It's done here conditionally to prevent mass-rebuilds.
-  checkPhase = lib.optionalString (isPy3k && stdenv.isDarwin) ''LANG="en_US.UTF-8" '' + (if isPy3k then ''
+  checkPhase = lib.optionalString (isPy3k && stdenv.isDarwin) ''LANG="en_US.UTF-8" LC_ALL="en_US.UTF-8" '' + (if isPy3k then ''
```

Comment:
[LANG](https://github.com/NixOS/nixpkgs/blob/f249f32f9e40c472c6085ddf61fb592e28c5118a/pkgs/development/interpreters/python/mk-python-derivation.nix#L93) is set by default now for python packages, I thought that had precedence over LC_*. Does this mean we should perhaps set both?

Reply:
I don't know, to be honest.  All I know is what i put above: without `LC_ALL` the build fails; with it set it succeeds.  I just followed the instructions on the website linked above :)